app-backend/backsplash-server/src/main/resources/application.conf

```diff
@@ -19,8 +19,11 @@ parallelism {
 }
 
 server {
-  timeoutSeconds = 15
-  timeoutSeconds = ${?BACKSPLASH_SERVER_TIMEOUT}
+  idleTimeoutSeconds = 63
```

Comment:
This needs to be greater than the ALB idle timeout and the http4s response header timeout.